src/DynamoCoreWpf/Views/GuidedTour/RatingControl.xaml

```diff
@@ -0,0 +1,40 @@
+﻿<UserControl x:Class="Dynamo.Wpf.Views.GuidedTour.RatingControl"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             xmlns:local="clr-namespace:Dynamo.Wpf.Views.GuidedTour"
+             mc:Ignorable="d" 
+             d:DesignHeight="300" d:DesignWidth="300">
+    <Grid>
+        <Grid.Resources>
+            <ControlTemplate x:Key="ratingtemplate" TargetType="{x:Type ToggleButton}">
+                <Viewbox>
+                    <Path Name="star" 
+                          Fill="Transparent" 
+                          StrokeThickness="1"
+                          Stroke="Black" 
+                          Data="F1 M 145.637,174.227L 127.619,110.39L 180.809,70.7577L 114.528,68.1664L 93.2725,5.33333L 70.3262,67.569L 4,68.3681L 56.0988,109.423L 36.3629,172.75L 91.508,135.888L 145.637,174.227 Z" />
+                </Viewbox>
+                <ControlTemplate.Triggers>
+                    <Trigger Property="IsChecked" Value="True">
+                        <Setter TargetName="star" Property="Fill" Value="#006EAF" />
+                    </Trigger>
+                </ControlTemplate.Triggers>
+            </ControlTemplate>
+        </Grid.Resources>
+        <Grid.ColumnDefinitions>
```

Comment:
Some comments in this file would be good here, e.g. why are there 5 of them, I guess because of the 5 stars

Reply:
I added some comments to the RatingControl.xaml in the next commit: https://github.com/DynamoDS/Dynamo/pull/11834/commits/84586419aa1ca8d940fcc13e6faff4ab4eba1968